<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAwardDialog</class>
 <widget class="QDialog" name="EditAwardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактировать награду</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Редактирование информации о награде</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="toolTip">
        <string>Введите название награды</string>
       </property>
       <property name="placeholderText">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Дата:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="toolTip">
        <string>Выберите дату получения награды</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="locationLabel">
       <property name="text">
        <string>Место проведения:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="locationLineEdit">
       <property name="toolTip">
        <string>Введите место проведения соревнования</string>
       </property>
       <property name="placeholderText">
        <string>Место проведения</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="sportLabel">
       <property name="text">
        <string>Вид спорта:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="sportComboBox">
       <property name="toolTip">
        <string>Выберите вид спорта</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="disciplineLabel">
       <property name="text">
        <string>Дисциплина:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="disciplineLineEdit">
       <property name="toolTip">
        <string>Введите дисциплину соревнования</string>
       </property>
       <property name="placeholderText">
        <string>Дисциплина</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="levelLabel">
       <property name="text">
        <string>Уровень соревнования:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="levelComboBox">
       <property name="toolTip">
        <string>Выберите уровень соревнования</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="placeLabel">
       <property name="text">
        <string>Занятое место:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="placeLineEdit">
       <property name="toolTip">
        <string>Введите занятое место</string>
       </property>
       <property name="placeholderText">
        <string>Занятое место</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="documentLabel">
       <property name="text">
        <string>Подтверждающий документ:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="documentLineEdit">
       <property name="toolTip">
        <string>Введите номер или название подтверждающего документа</string>
       </property>
       <property name="placeholderText">
        <string>Подтверждающий документ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditAwardDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditAwardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
